server/aws-lsp-codewhisperer/src/language-server/chat/chatSessionManagementService.ts

```diff
@@ -0,0 +1,45 @@
+import { CredentialsProvider } from '@aws/language-server-runtimes/server-interface'
+import { ChatSessionService, ChatSessionServiceConfig } from './chatSessionService'
+
+export class ChatSessionManagementService {
```

Comment:
Can this service be a singleton given there will probably be one chat server? 

For example for session management in inline code completions, we use a session manager singleton: https://github.com/aws/language-servers/blob/main/server/aws-lsp-codewhisperer/src/language-server/session/sessionManager.ts#L220

Reply:
Yeah that's a great idea! I will make this a singleton, thank you